types.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module "*.scss" {
+declare module "*.module.scss" {
```

Comment:
Same reasoning for not changing the script. The regex still applies.

Reply:
Makes sense, I'll switch them back

Reply:
I wonder if this is actually a good change because if you import a .scss file and don't get types for it, that would be a warning sign that you're importing CSS that won't be processed as a CSS module, because you didn't name it **.module.scss**.

In fact, I'm thinking we might even want to `declare module "*.scss" never` or something like that as a further guard against importing CSS files that aren't CSS modules.

Perhaps this is overcomplicating things, though.

Reply:
Certainly an interesting proposal. The two declarations may fight each other though (haven't tested, so could be wrong). But given this is only a problem because of the combination of this repo not compiling its own styles + Next being opinionated on the filename, it's not necessarily a long term restriction we require. 
KAIO for instance compiles its own styles, so this problem would never exist even if we were to name them all without `.module`.

Given we're looking into potential refactors to RTE + RTT for it to either live or be compatible with KAIO, it's better to leave implementing this for later (if at all) :)